pkg/env/env_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGet(t *testing.T) {
+	key := "ENV_TEST_TEST_GET"
+	value := "test value"
+	assert.NoError(t, os.Setenv(key, value))
```

Comment:
Very nit picky and the risk is pretty low but using `t.Setenv(...)` will automatically unset the env var after the test completes so that the env var doesn't affect other tests.